project-base/tests/App/Acceptance/acceptance/PaymentImageUploadCest.php

```diff
@@ -27,9 +28,13 @@ public function testSuccessfulImageUpload(AcceptanceTester $me, EntityEditPage $
         $me->amOnPage('/admin/payment/edit/1');
         $entityEditPage->uploadTestImage(self::IMAGE_UPLOAD_FIELD_ID, self::TEST_IMAGE_NAME);
         $me->clickByName(self::SAVE_BUTTON_NAME);
-        $me->seeTranslationAdmin('Payment <strong><a href="{{ url }}">{{ name }}</a></strong> modified', 'messages', [
-            '{{ url }}' => '',
-            '{{ name }}' => t('Credit card', [], 'dataFixtures', $me->getAdminLocale()),
-        ]);
+        $me->seeTranslationAdmin(
+            'Payment <strong><a href="{{ url }}">{{ name }}</a></strong> modified',
+            Translator::DEFAULT_TRANSLATION_DOMAIN,
+            [
+                '{{ url }}' => '',
+                '{{ name }}' => t('Credit card', [], 'dataFixtures', $me->getAdminLocale()),
```

Comment:
```suggestion
                '{{ name }}' => t('Credit card', [], Translator::DATA_FIXTURES_TRANSLATION_DOMAIN, $me->getAdminLocale()),
```
could we prepare all the translation domains we already use as constants in the Translator class?

Reply:
Thank you for mentioning, this was very fulfilling task